docs/development/local-setup.md

```diff
@@ -73,6 +75,20 @@ yarn serve
 
 You can now access the UI on http://localhost:8080/
 
+### 6. Login to the dashboard
+To login to the dashboard you can either configure oidc, or alternatively login using a token:
+
+To login using a token, first create a service account.
+`kubectl -n garden create serviceaccount dashboard-user``
```

Comment:
```suggestion
`kubectl -n garden create serviceaccount dashboard-user`
```

Reply:
however I'd rather prefer it as a code block as with my initial suggestion. It takes a bit more space but is better readable and a copy button is automatically provided by the UI

Reply:
ok